tests/integration/test_quantiles.py

```diff
@@ -112,12 +114,7 @@ def test_quantiles_1(str_method, axes, qin_arr, keepdims, overwrite_input):
     assert cu.all(q_out.shape == np_q_out.shape)
```

Comment:
No need to do `all` here (and similar checks below). Equality on tuples (which is what `.shape` returns) returns a scalar boolean.

Reply:
Done.